components/datadog/fakeintake/ecsFargate.go

```diff
@@ -14,41 +16,101 @@ const (
 	port          = 80
 )
 
-const (
-	DefaultFakeintakePort = 80
-)
+type Component struct {
```

Comment:
Perhaps it should be called `Instance`, what we what people to see externally is `fakeintake.Instance`.